s3prl/util/download.py

```diff
@@ -100,6 +101,60 @@ def _download_url_to_file(url, dst, hash_prefix=None, progress=True):
             os.remove(f.name)
 
 
+def _download_url_to_file_requests(url, dst, hash_prefix=None, progress=True):
```

Comment:
Hi @eric891224 , 

I think it would be nice to add your name on the top of this file.
After this is done, we can merge this PR.